python/n.py

```diff
@@ -88,36 +88,37 @@ def display(self, start_line, start_col, end_line, end_col):
 class TypeCheckError:
 	def __init__(self, token_or_tree, message):
 		if type(token_or_tree) is not lark.Token and type(token_or_tree) is not lark.Tree:
-			exit()
-			#raise TypeError("token_or_tree should be a Lark Token or Tree.")
+			raise TypeError("token_or_tree should be a Lark Token or Tree.")
 		self.datum = token_or_tree
 		self.message = message
 
 	def display(self, display_type, file):
 		output = ""
+		color = Fore.RED
```

Comment:
We can move this into the `if` statement because Python has weird variable scoping

Reply:
ehhh maybe I just thought this would be better and there wasn't any problems rn, I think the variable scoping only happens when there is a new function in the scope, not `if` statements